quick_start/my_text_classifier_bert.jsonnet

```diff
@@ -0,0 +1,48 @@
+local bert_model = "bert-base-uncased";
+
+{
+    "dataset_reader" : {
+        "type": "classification-tsv",
+        "tokenizer": {
+            "word_splitter": "bert-basic"
+        },
+        "token_indexers": {
+            "bert": {
+                "type": "bert-pretrained",
+                "pretrained_model": bert_model,
+            }
+        }
+    },
+    "train_data_path": "data/movie_review/train.tsv",
+    "validation_data_path": "data/movie_review/dev.tsv",
+    "model": {
+        "type": "simple_classifier",
+        "embedder": {
+            "allow_unmatched_keys": true,
+            "embedder_to_indexer_map": {
+                "bert": ["bert", "bert-offsets"]
+            },
+            "token_embedders": {
+                "bert": {
+                    "type": "bert-pretrained",
+                    "pretrained_model": bert_model,
+                    "top_layer_only": true,
+                    "requires_grad": false
+                }
+            }
+        },
+        "encoder": {
+            "type": "bert_pooler",
+            "pretrained_model": bert_model,
+            "requires_grad": false
+        }
+    },
+    "iterator": {
+        "type": "basic",
+        "batch_size": 8
+    },
+    "trainer": {
+        "optimizer": "adam",
```

Comment:
Is this the right optimizer for a bert model?  I know this is just an example, but we should probably have the right recommendations even in the example.

Reply:
I'm not sure if there's a "right" optimizer for a bert model (adam worked just fine for one of my past BERT projects) but the recommendation in the community, as far as I understand, is using bert_adam with a warmup using a low learning rate (e.g., 1.0e-5). I'll try this configuration and make sure the metrics are at least decent.